erlang/crypto_tests/sign.erl

```diff
@@ -0,0 +1,42 @@
+-module(sign).
+
+-export([cmd/0]).
+
+-include_lib("stdlib/include/assert.hrl").
+-include_lib("public_key/include/public_key.hrl"). 
+
+read_key(private, Name) ->
+    {ok, PemBin} = file:read_file(Name),
+    [RSAEntry] = public_key:pem_decode(PemBin),
+    Key = public_key:pem_entry_decode(RSAEntry),
+    Key;
+
```

Comment:
maybe add type annotation comments DDD:

Reply:
I'm just copy-pasting code - types are a blackbox

Reply:
if it works it's good enough :P

Reply:
also trust me knowing type doesn't make it better

3> Key = public_key:pem_entry_decode(RSAEntry, "abcd1234").
    #'RSAPrivateKey'{version = 'two-prime',
                 modulus = 1112355156729921663373...2737107,
                 publicExponent = 65537,
                 privateExponent = 58064406231183...2239766033,
                 prime1 = 11034766614656598484098...7326883017,
                 prime2 = 10080459293561036618240...77738643771,
                 exponent1 = 77928819327425934607...22152984217,
                 exponent2 = 36287623121853605733...20588523793,
                 coefficient = 924840412626098444...41820968343,
                 otherPrimeInfos = asn1_NOVALUE}